### PR TITLE
Support RTL language, Hebrew added

### PR DESCRIPTION
**Pull request by McGiverGim**
This is a WORK IN PROGRESS.

I have pushed the PR in an early stage to see if my solution is acceptable.

Some languages are RTL (right-to-left) like hebrew or arab. The configurator is not ready for such languages.

The ideal solution would be to rewrite the design (or at least all the css stuff) thinking in both type of languages (RTL and LTR), but the configurator is working, and maybe changing something in one part, breaks other part that is not visible at first, so I prefer to search a solution that at least didn't broke anything of the actual configurator.

The solution is easy:
- When localizing the page, detect if the language is RTL or LTR.
- If RTL, add the `dir="rtl"` attribute to the page, and then load a CSS file, called `reset-rtl.css`
- This new CSS file, overwrites some values defined by all the others CSS, to align the things correctly for a RTL language.

The objective is not to have a "perfect" RTL system (at least not by the moment), but have a system that is acceptable for this kind of languages.

Before I make all the work. **Is this solution acceptable?** It will not broke anything, the developers can forget about RTL, and the worst that can happen is that this file gets outdated and some things appear "ugly" in RTL languages.

### CODE REVIEW

**Comment by mikeller** [comment]
I think this will need some testing to find out if it can be made acceptable. One key criterion for me is that the impact / extra work that is required to support this for contributors wanting to do things like adding new UI elements should be minimal, and it should fail to safe defaults if a contributor is not aware of the extra requirements (this potentially includes CSS required to format the new UI elements).

**Comment by McGiverGim** [comment]
Yes, that was my idea too. Any new thing done by a developer is not impacted by any of this PR, only if a RTL language is used.
In this PR I have formatted four tabs to support RTL as example and I haven't touched the standard code.

File: src/css/main.css [hunk 1]
```diff
@@ -453,7 +453,7 @@ input[type="number"]::-webkit-inner-spin-button {
     width: 110px;
     opacity: 0.15;
     background-size: 80%;
-    box-shadow: inset 0 0 5px #000000;
+    //box-shadow: inset 0 0 5px #000000;
```
Comment (mikeller) [inline src/css/main.css]: This should be removed altogether.

**Comment by mikeller** [comment]
Is this ready to go, or is there more to come to fully enable RTL support?

File: src/css/main.css (hunk 1 above)
Comment (McGiverGim) [reply src/css/main.css]: Yes, I will do, this is only a draft of the final result :)

**Comment by McGiverGim** [comment]
This is a work that is only beginning... I'm working with the hebrew moderator, modifying the tabs when the translations are done. It will last some weeks more, and the reset css will grow a lot.

The only objective was to show the way that I'm doing it. The ideal way is to modify completely the structure of the actual application. But this is a dangerous work, a lot of things could be wrong, so I prefered, at least by the moment use this solution: a reset css only for RTL languages.

**Comment by mikeller** [comment]
@McGiverGim: Agreed on your course of action. And agreed about it being a minefield (think of things like vertical labels).

**Comment by McGiverGim** [comment]
The development of this is stopped. I close this PR and if it ends at some moment I will reopen it again.

**Pull request closed without merging**